webpack.config.js

```diff
@@ -0,0 +1,40 @@
+// webpack.config.js
+const path = require('path');
+const Dotenv = require('dotenv-webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = (env) => ({
+  entry: './src/index.tsx',
+  output: {
+    path: path.resolve(__dirname, './build'),
+    filename: 'bundle.js'
+  },
+  devServer: {
+    static: {
+      directory: path.join(__dirname, 'build')
+    },
+    port: 3000
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, 'public', 'index.html')
+    }),
+    new Dotenv({
+      path: `./.env.${env}`
+    })
```

Comment:
Interesting approach. Have you seen it somewhere or it is your idea?

Reply:
I used a similar approach to solve another problem. I just remembered that and tried to find a way to implement it with webpack and react. And found the dotenv-webpack pack.